sqlgen/db.go

```diff
@@ -92,6 +92,23 @@ func NewDB(conn *sql.DB, schema *Schema) *DB {
 	return db
 }
 
+func (db *DB) BaseCount(ctx context.Context, query *BaseCountQuery) (*int64, error) {
```

Comment:
Ryan's opinion.  Return (int64, error).  Removes pointer handling from the non-error case.